os/arch/arm/src/imxrt/imxrt_clockconfig.c

```diff
@@ -69,29 +69,101 @@
  * Definitions
  ******************************************************************************/
+/* USB PHY condfiguration */
+#define BOARD_USB_PHY_D_CAL (0x0CU)
+#define BOARD_USB_PHY_TXCAL45DP (0x06U)
+#define BOARD_USB_PHY_TXCAL45DM (0x06U)
+
+#if defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
+#define IMXRT_USBPHY_BASE USBPHY_BASE
+#elif defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
+#define IMXRT_USBPHY_BASE USBPHY1_BASE
+#endif            
+
+typedef struct _usb_phy_config_struct {
+	uint8_t D_CAL;				/* Decode to trim the nominal 17.78mA current source */
+	uint8_t TXCAL45DP;			/* Decode to trim the nominal 45-Ohm series termination resistance to the USB_DP output pin */
+	uint8_t TXCAL45DM;			/* Decode to trim the nominal 45-Ohm series termination resistance to the USB_DM output pin */
+} usb_phy_config_struct_t;
+
 /*******************************************************************************
  * Variables for BOARD_BootClockRUN configuration
  ******************************************************************************/
 #if defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
 const clock_enet_pll_config_t enetPllConfig_BOARD_BootClockRUN = {
-    .enableClkOutput500M = true,
+	.enableClkOutput500M = true,
 };
 #endif
 const clock_arm_pll_config_t armPllConfig_BOARD_BootClockRUN = {
-    .loopDivider = 100, /* PLL loop divider, Fout = Fin * 50 */
-    .src = 0,           /* Bypass clock source, 0 - OSC 24M, 1 - CLK1_P and CLK1_N */
+	.loopDivider = 100, /* PLL loop divider, Fout = Fin * 50 */
+	.src = 0,           /* Bypass clock source, 0 - OSC 24M, 1 - CLK1_P and CLK1_N */
 };
 const clock_sys_pll_config_t sysPllConfig_BOARD_BootClockRUN = {
-    .loopDivider = 1, /* PLL loop divider, Fout = Fin * ( 20 + loopDivider*2 + numerator / denominator ) */
-    .numerator = 0,   /* 30 bit numerator of fractional loop divider */
-    .denominator = 1, /* 30 bit denominator of fractional loop divider */
-    .src = 0,         /* Bypass clock source, 0 - OSC 24M, 1 - CLK1_P and CLK1_N */
+	.loopDivider = 1, /* PLL loop divider, Fout = Fin * ( 20 + loopDivider*2 + numerator / denominator ) */
+	.numerator = 0,   /* 30 bit numerator of fractional loop divider */
+	.denominator = 1, /* 30 bit denominator of fractional loop divider */
+	.src = 0,         /* Bypass clock source, 0 - OSC 24M, 1 - CLK1_P and CLK1_N */
 };
 const clock_usb_pll_config_t usb1PllConfig_BOARD_BootClockRUN = {
-    .loopDivider = 0, /* PLL loop divider, Fout = Fin * 20 */
-    .src = 0,         /* Bypass clock source, 0 - OSC 24M, 1 - CLK1_P and CLK1_N */
+	.loopDivider = 0, /* PLL loop divider, Fout = Fin * 20 */
+	.src = 0,         /* Bypass clock source, 0 - OSC 24M, 1 - CLK1_P and CLK1_N */
 };
 
+const usb_phy_config_struct_t PhyConfig = {
+	BOARD_USB_PHY_D_CAL, BOARD_USB_PHY_TXCAL45DP, BOARD_USB_PHY_TXCAL45DM,
+};
+
+/****************************************************************************
+ * Function: imxrt_edma_getinstance
```

Comment:
Function name is wrong. Please change. 
Also, description seems to be incomplete. Please check it.